### PR TITLE
MEPTS-755: APSS Report - Add E3 Indicator

### PR DESCRIPTION
**Pull request by amosh101**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java [hunk 1]
```diff
@@ -437,11 +437,115 @@ public CohortDefinition getE2() {
...
+            + "                      INNER JOIN obs o ON e.encounter_id = o.encounter_id     "
+            + "                      INNER JOIN obs o2 ON e.encounter_id = o2.encounter_id     "
+            + "                      WHERE     "
+            + "                          p.patient_id = e.patient_id     "
+            + "                          AND p.voided = 0     "
+            + "                              AND e.voided = 0     "
+            + "                              AND o.voided = 0     "
+            + "                              AND e.encounter_type = ${61}     "
+            + "                              AND e.location_id = :location     "
+            + "                              AND o.concept_id = ${24011}     "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:537]: I cant see this in the specs

**Review by psmatsinhe — COMMENTED** [review]
@amosh101 Please have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java (hunk 1 above)
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:537]: https://docs.google.com/document/d/159ZGr5KJcw8tkb-FXKncBxVgplYJFMOU/edit This is the spec

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**